.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
I believe ts-pre-commit shall manage those, and put those to .gitignore, see e.g. gitignore in ts_mtaircompressor. Remove this and isort&mypy files?

Reply:
It is all explained here: https://tssw-developer.lsst.io/development-guidelines/tools/pre_commit.html#ts-pre-commit-conf